src/components/calendar/fCCalendar.stories.tsx

```diff
@@ -25,34 +16,54 @@ export default {
   },
 }
 
+meta.argTypes = {
+  ...meta.argTypes,
+  events: {
+    table: {
+      type: {
+        summary: 'Array of CalendarEvent.\n',
+        detail:
+          'Each CalendarEvent has the following fields:\n' +
+          '  start: ISO8601 string representation of start date and time of the event\n' +
+          '  end: ISO8601 string representation of end date and time of the event\n' +
+          '  title: Title or Name of the event.\n' +
+          '  location: Optional physical location or online link where event is happening\n' +
+          '  description: Optional information about the event.\n' +
+          '  isAllDay: Optional boolean value indicating whether the event lasts the entire day.',
```

Comment:
Should we add '?' to optional props and add data types to all of the fields so that it's more clear to people? You mentioned string and boolean types in some fields, but types for some fields are missing

Reply:
This is not the class definition, hence no `?`.  I'm hoping this is a temporary workaround and we can have types documented somewhere. I thought the descriptions were clear enough but can update them if required. Please feel free to add suggestions

Reply:
Can we add `(string value)` next to `physical location`. I was wondering if you mean address string or the combination of longitude and latitude. 
```suggestion
          'Each CalendarEvent has the following fields:\n' +
          '  start: ISO8601 string representation of start date and time of the event\n' +
          '  end: ISO8601 string representation of end date and time of the event\n' +
          '  title: Title or Name of the event.\n' +
          '  location: Optional physical location(string value) or online link where event is happening\n' +
          '  description: Optional information about the event.\n' +
          '  isAllDay: Optional boolean value indicating whether the event lasts the entire day.',
```